application.py

```diff
@@ -36,6 +36,9 @@ def calculate():
     if operation == "custom_log_base":
         base = float(request.form["number_2"])
         result = math.log(number_1, base)
+    if operation == "modulus":
+        divider = float(request.form["number_2"])
+        result = math.modf(number_1, divider)
```

Comment:
I actually didn’t think of using the math.modf function as I was thinking of using “%”, like:

```python
i = 5 % 3 # 2
```

But I think your method is more appropriate.

Reply:
After testing this code, I found that the math.modf function did not work as expected and I changed your code to use the Python modulus operator (%)

See #59 for my changes